docs/sphinx/dev_guide/contributing.rst

```diff
@@ -138,9 +159,9 @@ Testing Pull Requests from Branches in Forked Repositories
 
 Due to LLNL security policies and RAJA project policies, only a PR created
 by someone on the RAJA core development team will be run automatically
-through all RAJA CI tools. A PR made from a forked repository will not. 
-Specifically, Gitlab CI on internal LLNL platforms and Travis CI will only
-be run on PRs that are made on branches within the GitHub RAJA repository.
+through all RAJA CI tools. In particular, a PR made from branch on a forked 
+repository will not. Gitlab CI on internal LLNL platforms and Travis CI will 
```

Comment:
(I thought this was worded oddly, maybe this would be better?)
```suggestion
through all RAJA CI tools. In particular, a PR made from a branch on a forked 
repository will not automatically trigger the RAJA CI. Gitlab CI on internal LLNL platforms and Travis CI will 
```

Reply:
Got it.